__tests__/index.test.js

```diff
@@ -21,4 +22,24 @@ test('GET: 200 sends and array of topics', () => {
         })
     })
 })
+})
+
+
+
+//endpoints test
+
+describe('/api/', () => {
+    test('GET:200 responds with and object of API endpoints', () => {
+    return request(app)
+    .get('/api')
+    .expect(200).then(({body}) => {
+    
+    expect(body).toEqual(endpoints);
+
+    for (let endpoint in body) {
+        expect(typeof body[endpoint].description).toBe('string');
```

Comment:
since we're doing the above we wouldn't need to test in this way as we've expected it should look how we expect.

Reply:
I'll remove it, thanks